src/cpu/dyn_cache.h

```diff
@@ -726,10 +777,10 @@ static void cache_init(bool enable) {
 		cache_initialized = true;
 		if (cache_blocks == NULL) {
 			// allocate the cache blocks memory
-			cache_blocks = (CacheBlock *)malloc(CACHE_BLOCKS * sizeof(CacheBlock));
+			cache_blocks = (CacheBlock *)malloc(cache_blocks_sz);
 			if (!cache_blocks)
 				E_Exit("Allocating cache_blocks has failed");
-			memset(cache_blocks, 0, sizeof(CacheBlock) * CACHE_BLOCKS);
+			memset(cache_blocks, 0, cache_blocks_sz);
```

Comment:
Might as well expand `sz` to the full unit name just for consistency. (otherwise.. the other parts could be collapsed too.. `cch_blks_sz` :-)

`cache blocks size` tells me it's the size of the cache blocks (a bit tricks.. it's all the blocks, not one; someone skimming the code might not catch that nuance).  Maybe, `total_cache_bytes` or something to that effect?